docs/source/developer_guides/contributing.md

```diff
@@ -14,79 +14,70 @@ rendered properly in your Markdown viewer.
 
 -->
 
-# Contributing to PEFT
+# Contribute to PEFT
 
-We are happy to accept contributions to PEFT. If you plan to contribute, please read this document to make the process as smooth as possible.
+We are happy to accept contributions to PEFT. If you plan to contribute, please read this to make the process as smooth as possible.
 
 ## Installation
 
-The installation instructions can be found [here](https://huggingface.co/docs/peft/install). If you want to provide code contributions to PEFT, you should choose the "source" installation method.
+For code contributions to PEFT, you should choose the ["source"](../install#source) installation method.
 
-If you are new to creating a pull request, follow [these instructions from GitHub](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request).
+If you are new to creating a pull request, follow the [Creating a pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request) guide by GitHub.
 
-## Running tests and code quality checks
+## Tests and code quality checks
 
-Regardless of the type of contribution (unless it’s only about the docs), you should run tests and code quality checks before creating a PR to ensure that your contribution doesn’t break anything and follows the standards of the project.
+Regardless of the contribution type (unless it’s only about the docs), you should run tests and code quality checks before creating a PR to ensure your contribution doesn’t break anything and follows the project standards.
 
-We provide a Makefile to facilitate those steps. Run the code below for the unit test:
+We provide a Makefile to execute the necessary tests. Run the code below for the unit test:
 
 ```sh
 make test
 ```
 
-Run one of the following to either check or check and fix code quality and style:
+Run one of the following to either only check or check and fix code quality and style:
 
 ```sh
 make quality  # just check
 make style  # check and fix
 ```
 
-
-Running all the tests can take a couple of minutes. Therefore, during development, it can be useful to run only those tests specific to your change:
+Running all the tests can take a couple of minutes so during development, it can be useful to only run tests specific to your change:
 
 ```sh
 pytest tests/ -k <name-of-test>
 ```
 
-This should finish much quicker and allow faster iteration. Before creating the PR, however, please still run the whole test suite, as some changes can inadvertently break tests that at first glance are unrelated.
+This should finish much quicker and allow faster iteration. However, you should still run the whole test suite as some changes before creating a PR because it can inadvertently break tests that at first glance are unrelated.
```

Comment:
line 50 redundant phrase "as some changes" (?)
Proposed: "However, you should still run the whole test suite ~~as some changes~~ before creating a PR because it ..."